### ISSUE TITLE
[Bug] plq_Ridge_Classifier raises Exception for 'squared SVM' 

### ISSUE THREAD

**Issue by BenViolet6** [softmin/ReHLine-python#35]
## Bug Description
I am trying to use `plq_Ridge_Classifier` with the `'squared SVM'` loss function. However, the code raises an exception in `_make_loss_rehline_param`, stating that the loss function is not supported.

It seems the string matching for the loss name might be failing or the wrapper doesn't handle this specific string format.

## Expected Behavior
The model should successfully initialize and fit the data using the specified loss function ('squared SVM').

## Actual Behavior
The code raises an Exception during the .fit() call inside _make_loss_rehline_param. The error message states: "Sorry, ReHLine currently does not support this loss function", suggesting the loss name is not being parsed correctly.

## Steps to Reproduce
1. Use make_classification to generate data (n=10000, d=5).
2. Standardize the data using StandardScaler.
3. Initialize plq_Ridge_Classifier with loss={'name': 'squared SVM'}.
4. Call clf.fit(X, y).

## Minimal Code Example

```python
# Minimal reproducible example
import numpy as np
from rehline import ReHLine

from sklearn.datasets import make_classification
from sklearn.preprocessing import StandardScaler
from rehline import plq_Ridge_Classifier
import numpy as np

# 1. Simulate data 
scaler = StandardScaler()
n, d = 10000, 5
X, y = make_classification(n_samples=n, n_features=d, random_state=42)
X = scaler.fit_transform(X)

# 2. Solve SVM via plq_Ridge_Classifier
# This line causes the crash
clf = plq_Ridge_Classifier(loss={'name': 'squared SVM'}, C=1.0)
clf.fit(X=X, y=y)
```

## Environment

- OS: Linux / 6.6.113+
- Python version: 3.12.12
- ReHLine version: Not found (or check pip show rehline)
- NumPy version: 2.0.2
- SciPy version: 1.16.3
- scikit-learn version: 1.6.1
